spec/services/quotes_service_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe QuotesService, type: :service do
+  describe 'Class Methods', :vcr do
+    describe '.quotes(:keyword)' do
+      it 'returns an index of all Quotes related to keyword' do
+        response = QuotesService.quotes('life happiness')
+        expect(response[:results]).to be_an Array
+        response[:results].each do |quote|
+          quote_attributes(quote)
+        end
+      end
+    end
+  end
+end
```

Comment:
check for totalCount too since your code relies on that being present in a particular spot.